src/Rubric/RubMoveUpAction.class.st

```diff
@@ -0,0 +1,19 @@
+Class {
+	#name : 'RubMoveUpAction',
+	#superclass : 'RubCursorAction',
+	#category : 'Rubric-Editing-Core',
+	#package : 'Rubric',
+	#tag : 'Editing-Core'
+}
+
+{ #category : 'testing' }
+RubMoveUpAction >> isForward [
+	
+	^ true.
+]
```

Comment:
I think these should be the other way around, right? `RubMoveUpAction>>isForward -> false` and `RubDownUpAction>>isForward -> true`